### PR TITLE
Gatsby UX Capture plugin and example

### PR DESCRIPTION
**Pull request by sergeychernyshev**
Gatsby integration plugin:
- injects inline copy of UXCapture library
- fires `UXCapture.startTransition()` before routes change

Example Gatsby site with UX Capture integration:
- injects library initialization using `<UXCaptureCreate />`
- injects marks into Gatsby-generated static HTML files
- works in both `page view` and `interactive view` modes (has 3 sample views)
- does NOT include mark and measure visualizations

Pivotal ticket: https://www.pivotaltracker.com/story/show/169297083

### CODE REVIEW

File: packages/gatsby-plugin-ux-capture/.gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/*.js
+!index.js
+yarn.lock
```
Comment (cranberyxl) [inline packages/gatsby-plugin-ux-capture/.gitignore]: Why ignore the yarn.lock?

Comment (sergeychernyshev) [reply packages/gatsby-plugin-ux-capture/.gitignore]: Probably a default from plugin repo I copied. In this case though, yarn.lock is in the root of the monorepo anyway.

Comment (sergeychernyshev) [reply packages/gatsby-plugin-ux-capture/.gitignore]: I'll remove it.

**Comment by sergeychernyshev** [comment]
Hmm, looks like some issues with onMark handler in the code - didn't see it before... checking.

I'll probably refactor it a tiny bit as I did in nextjs PR to remove some extraneous abstractions.

**Comment by sergeychernyshev** [comment]
Ignore my comment about `onMark` - it's TypeScript warnings.
Probably a separate ticket to define TypeScript annotations for the whole UX Capture.

**Comment by sergeychernyshev** [comment]
Made some minor adjustments, but for some reason getting oMark errors when I run `gatsby develop` and no errors if I run `gatsby build && gatsby serve`.

**Pull request merged**